Sources/Rownd/Models/PasskeyCoordinator.swift

```diff
@@ -221,7 +224,7 @@ class PasskeyCoordinator: NSObject, ASAuthorizationControllerPresentationContext
         authController.performRequests()
     }
 
-    func authorizationController(controller: ASAuthorizationController, didCompleteWithAuthorization authorization: ASAuthorization) {
+    public func authorizationController(controller: ASAuthorizationController, didCompleteWithAuthorization authorization: ASAuthorization) {
```

Comment:
Why is this function (along with authorizationController() and presentationAnchor) being made public?

Reply:
The PasskeyCoordinator class itself is internal so these endpoints should be safe. I just removed the public from these func because it's not required.